crates/revm/Cargo.toml

```diff
@@ -46,14 +46,13 @@ dev = [
     "optional_eip3607",
     "optional_gas_refund",
 ]
+ethersdb = ["tokio", "futures", "ethers-providers", "ethers-core", "hex"]
+k256 = ["revm_precompiles/k256_ecrecover"]
 memory_limit = []
 no_gas_measuring = []
 optional_block_gas_limit = []
 optional_eip3607 = []
 optional_gas_refund = []
-std = ["bytes/std", "num_enum/std", "rlp/std"]
 secp256k1 = ["revm_precompiles/secp256k1"]
-k256 = ["revm_precompiles/k256_ecrecover"]
-web3db = []
```

Comment:
Removed deprecated feature flag, as it will cause `cargo t --all-features` to fail